front-end/src/components/MainPage.test.js

```diff
@@ -12,6 +13,14 @@ describe('MainPage tests', () => {
         </Provider>
     );
 
+    beforeEach(() => {
+        store.dispatch(enable())
+    });
```

Comment:
(This goes for other test files as well) We probably do not want to enable this before each test but instead before certain tests. We should also explicitly add test cases for when analysis page is enabled and disabled.